docs/personalization/api_reference/tracking_api.md

```diff
@@ -358,7 +358,8 @@ Sending Rendered events causes as many requests as recommendations to be display
 Example of a recommendation response:
 
 ``` json
-"recommendationItems": [
+{
+  "recommendationItems": [
```

Comment:
We should provide a full response here, in an external file, not an extract. And, if needed, highlight lines related to the topic.